.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "java.configuration.updateBuildConfiguration": "interactive"
+}
```

Comment:
Please add this file into **.gitignore** and remove from there.

Reply:
The changes have been reverted, in the future I'll use a separate branch for my own purposes.